Source/cmake/WebKitCompilerFlags.cmake

```
@@ -136,6 +136,11 @@ if (DEVELOPER_MODE AND DEVELOPER_MODE_FATAL_WARNINGS)
     endif ()
 endif ()
 
+if (DEVELOPER_MODE)
+    # This lets us get good backtraces, in particular when using JSC_useGdbJITInfo=1.
+    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer)
```

So I know I originally suggested `-mno-omit-leaf-frame-pointer`, but:

```suggestion
    WEBKIT_PREPEND_GLOBAL_COMPILER_FLAGS(-fno-omit-frame-pointer)
```

See: [thread](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/thread/IRII7NCM7GDHCV6USEJYANK3XDGOMPEV/#43YIN56IVUU2PV2HQZO2SCL5O3LEYXMW).